KK_ButtPlugin/ButtplugWsClient.cs

```diff
@@ -30,15 +33,18 @@ public void Open()
 
         public void Close()
         {
+            Devices.Clear();

```

Comment:
Don't see why. This is only called when you quit the game.

Reply:
I'm also calling close when clicking the "connect" button just as a precaution to force a clean state.

Reply:
I guess instead of clearing the list, since we make a new one instead of updating I can just replace it on open.